sm/json_sm/src/messages.rs

```diff
@@ -155,6 +156,33 @@ impl SoftwareUpdateRequest {
 
         updates
     }
+
+    // pub fn updates_for_downloadable(&mut self) -> Vec<SoftwareModuleUpdate> {
```

Comment:
Don't forget to use or remove this dead code before this PR is declared ready for review.